client/src/components/AddCard/AddCardForm/CreditCard/CreditCard.tsx

```diff
@@ -26,9 +26,14 @@ const CreditCard = ({ card, setValues, handleSubmit, values }: Props): JSX.Eleme
   return (
     <Box padding="20px" marginRight={2} border={2} borderColor="#dddddd" borderRadius={10} width={340} height={190}>
       <Box height="36px" marginBottom={theme.spacing(0.4)} display="flex" justifyContent="space-between">
-        <img src={logos[card.card.brand as keyof logosModel]} className={classes.logo} />
+        <img alt="brand" src={logos[card.card.brand as keyof logosModel]} className={classes.logo} />
         <Radio
-          checkedIcon={<CheckIcon style={{ color: '#ffffff', backgroundColor: '#f14140', borderRadius: '1.5rem' }} />}
+          checkedIcon={
+            <CheckIcon
+              className={classes.checkedIcon}
+              style={{ color: '#ffffff', backgroundColor: '#f14140', borderRadius: '1.5rem' }}
```

Comment:
Please remove these styles as we are already using a CSS class to apply them

Reply:
Oh, thanks for reminding me. Done.